index.html

```diff
@@ -2003,6 +2009,70 @@ <h3>Agility vs. Hardening</h3>
+YAML, and other compatible syntaxes without having to recreate the cryptographic
+proof for every syntax.
+        </p>
+        <p>
+Being able to express the same digital signature across a variety of syntaxes is
+beneficial because systems often have native data formats with which they
+operate. For example, some systems are written against JSON data, while others
+are written against CBOR data. Without transformation, systems that process
+their data as CBOR internally are required to store the digitally signed data
+structures as JSON (or vice-versa). This leads to double-storing data and can
+lead to extra security attack surface if the unsigned representation stored in
+databases accidentally deviates from the signed representation. By using
+transformations, the digital proof can live with the native data format to
+prevent undetectable database drift over time.
+        </p>
+        <p>
+This specification is designed to avoid requiring the duplication of signed
+information by utilizing data transformations. Application developers are urged
+to work with cryptographically protected data in the native data format for
+their application and not separate storage of cryptographic proofs from the data
+being protected. Developers are also urged to regularly confirm that the
+cryptographically protected data has not been tampered with as it is written to
+and read from application storage.
+        </p>
+        <p class="issue"
+          title="Collision-resistant canonicalization requirements">
+The VCWG is seeking feedback on normative language that cryptographic suite
+implementers need to follow to ensure that they do not utilize data
+transformation mechanisms that can map to the same output. That is, given
+the different inputs for canonicalization scheme #1 and canonicalization
+scheme #2, they must not produce the same output value. As an analogy, this is
+the same requirement for cryptographic hashing mechanisms and is why those
+schemes are designed to be collision resistant. Cryptographic canonicalization
+mechanisms have the same requirement. At present, this isn't a problem because
+the three expected canonicalization schemes, the Universal RDF Dataset
+Canonicalization Algorithm 2015 [[?URDNA2015]], JSON Canonicalization Scheme
+[[?RFC8785]], and a theoretical future base-encoding canonicalization have
+entirely different outputs.
+        </p>
+        <p class="issue"
+          title="Avoiding the pitfalls of XML Canonicalization">
+The VCWG is seeking feedback on whether to explain why modern canonicalization
+schemes are simpler than the far more complex XML Canonicalization schemes of
+the early 2000s. Some readers seem to be under the impression that all
+canonicalization is difficult and has to be avoided at all costs. The WG would
+like to understand if a section explaining how some simpler data syntaxes (such
+as JSON) are easier to canonicalize than more complex data syntaxes (such as
+XML).
```

Comment:
```suggestion
canonicalization is difficult and has to be avoided at all costs (including costs
to application developers). The WG would like to understand if a section
explaining how some simpler data syntaxes (such as JSON) are easier to
canonicalize than more complex data syntaxes (such as XML).
```

Reply:
`explaining how some simpler` in line 2070 (unedited) or line 2071 (@dlongley's edit) should be `explaining why some simpler`